modules/special-rules/pages/flak.adoc

```diff
@@ -1,2 +1,63 @@
 = Flak
 
+These units are specialised to shoot down enemy flyers and drop pods.
+Some use very sophisticated targeting devices and fast, accurate weapons systems.
+Others simply fill the air with so many munitions that they can hardly fail to get some hits.
+
+---
+
+{flak} units can attack flyers in one of two different ways, depending on what orders you give to their detachment.
+
+[NOTE]
+.Flak and drop pods
+====
+{flak} units may also shoot at drop pods, as if they were flyers.
+For your rolls to hit, use the Armour value of the transported unit.
+====
+
+.Flak orders
+For optimum effectiveness, {flak} units and their detachment need to be at a suitable state of readiness.
+We represent this state with special orders called Flak.
+
+. If you wish to place any detachments on Flak orders then you must do this in the Start phase.
++
+[TIP]
+====
+You can place a detachment on both Flak orders and Preparatory Bombardment orders at the same time in the Start phase, if you wish.
+You must declare both at the same time.
+
+In this case, you will be able to make a Preparatory Bombardment with {artillery} units in the detachment, and make Flak attacks with {flak} units.
+====
+. Place the {special-orders} marker to indicate that the detachment is on Flak orders.
+. You can use these units to shoot at flyers in the movement phase.
+ * You may not move the units in the Movement phase, nor use them to shoot in the Shooting phase.
+ * Treat the other units in the detachment as though they are on Overwatch orders -- so you can move them up to 5cm in the Movement phase and re-roll misses when you shoot with them in the Shooting phase.
+. A {flak} unit can shoot at enemy flyers if any part of their flight path is within the {flak} unit's full Range value:
+ * Roll one D6 for each point of the {flak} unit's Firepower value.
+ Each result that is equal to or higher than the flyer's Armour value scores a hit on the flyer.
+ You can split the firepower of the {flak} unit between several flyers if you wish.
+ * Once you have rolled a D6 for each point of the unit's Firepower, the unit cannot shoot for the rest of the turn.
+ * You can apply the effects of super-heavy weapons. +
+.Flak, with a super-heavy weapon
+====
+The Eldar Fire Prism has a single {macro-weapon} attack.
+This means that you will roll one die, and you'll hit on a result of 4+ — regardless of the flyer's Armour value.
+====
+
+.Any other orders — Flak Snap-Fire attack
+Apply these rules to the unit's Snap Fire attacks against flyers:
+
+* Use the {flak} unit's full Range value, instead of the usual 10cm.
++
+NOTE: You may not use super-heavy weapon effects — such as {macro-weapon} — for Flak Snap-Fire attacks.
```

Comment:
Actually, don't need the word Flak at the end there after all do we. SHW effects aren't allowed on regular Snap-Fire attacks either, are they?